.github/workflows/TagBot.yml

```diff
@@ -1,9 +1,12 @@
 name: TagBot
 on:
-  schedule:
-    - cron: 0 * * * *
+  issue_comment:
+    types:
+      - created
+  workflow_dispatch:
 jobs:
   TagBot:
+    if: github.event_name == 'workflow_dispatch' || github.actor == 'JuliaTagBot'
```

Comment:
I *think* it should be `github.actor == 'beacon-buddy'`. Once we know it's working, we could use https://github.com/bcbi/MassInstallAction.jl to PR all our repos, or just do it manually as it comes up.

Reply:
This is a public repository that we're already using public JuliaRegistrator/JuliaTagBot on (unless I misunderstand what you're getting at)

Reply:
Oops, yup, sorry